shub/deploy.py

```diff
@@ -128,14 +128,15 @@ def _url(endpoint, action):
 
 
 def _upload_egg(endpoint, eggpath, project, version, auth, verbose, keep_log,
-                stack=None, requirements_file=None):
+                stack=None, requirements_file=None, eggs=None):
+    eggs = eggs or []
     data = {'project': project, 'version': version}
     if stack:
         data['stack'] = stack
-    files = {'egg': ('project.egg', open(eggpath, 'rb'))}
+    files = [('egg', open(path, 'rb')) for path in eggs]
```

Comment:
As a proposal: maybe catch `IOError` here and raise `ShubException` with some nice message, instead of :point_down: ?

``` 
 File "/home/kris/ws/schub/sh_tools/shub/shub/deploy.py", line 136, in _upload_egg
    files = [('egg', open(path, 'rb')) for path in eggs]
IOError: [Errno 2] No such file or directory: 'boto/dist/boto-2.42.0-py2.7.egg1'```